src/localization/locale/en.ts

```diff
@@ -88,6 +88,8 @@ export const en = {
   info: {
     artifact: 'Artifact',
     noResults: 'No tracked results',
+    flowRunResults: 'Flow run',
+    taskRunResults: 'Task runs',
```

Comment:
Nit: As we start adding more of these we need to be careful about naming. If we ever add a second language the names will be important. I'd literally name these `flowRun` and `taskRuns` as these are pretty generic labels and aren't restricted to results.